lib/statsd/instrument.rb

```diff
@@ -97,7 +97,7 @@ def statsd_measure(method, name, deprecated_sample_rate_arg = nil, deprecated_ta
         define_method(method) do |*args, &block|
           key = StatsD::Instrument.generate_metric_name(name, self, *args)
           prefix ||= StatsD.prefix
-          StatsD.measure(
+          StatsD.measure( # rubocop:disable StatsD/MeasureAsDistArgument, StatsD/MetricPrefixArgument
```

Comment:
Not sure it makes sense to enable those cops on statsd-instrument itself.

Reply:
I like it, because we make sure the rules actually work this way.

This particular exception can :fire: when we do the next major release.